examples/jsm/lights/RectAreaLightUniformsLib.js

```diff
@@ -25,6 +25,59 @@ import {
 // by Eric Heitz, Jonathan Dupuy, Stephen Hill and David Neubelt
 // code: https://github.com/selfshadow/ltc_code/
 
+// Convert float32 array value to float16 stored in uint16 array value.
+// Source: http://gamedev.stackexchange.com/questions/17326/conversion-of-a-number-from-single-precision-floating-point-representation-to-a/17410#17410
+
+var floatView = new Float32Array( 1 );
+var int32View = new Int32Array( floatView.buffer );
+
+/* This method is faster than the OpenEXR implementation (very often
+* used, eg. in Ogre), with the additional benefit of rounding, inspired
+* by James Tursa?s half-precision code. */
+
+function toHalf ( val ) {
```

Comment:
Copying this code of now is fine. When `examples/js` is gone at the end of the year, I suggest to create a new utility class where this function should be moved.

Reply:
OK I'll add a comment saying this function is a duplicate.